static/opensearch.xml

```diff
@@ -0,0 +1,7 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+	<ShortName>Charmhub</ShortName>
+	<Description>Charmhub - The Open Operator Collection</Description>
+	<InputEncoding>[UTF-8]</InputEncoding>
```

Comment:
I think the square brackets are not needed here and in the next line.

Reply:
Done